### DISCUSSION TITLE
Using cycle routes in Mountainbike and Bike profiles

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#596] [Q&A] [answered]
Since few weeks I try to use Cruiser for my bike tours - planning in Komoot, navigating with Cruiser. 
Normally, the Bike profile would be the right choice for this, because these are simple bike tours, guided for older people. However, I have now noticed that some cycle routes are not used in the Bike profile and have switched to the Mountain Bike profile, because there I can define "use cycle routes": 
<img width="200" src="https://github.com/user-attachments/assets/8555b8f9-887e-4dd8-a4f6-cfac5e16417f"/> . . . <img width="200" src="https://github.com/user-attachments/assets/9986cec5-9959-4101-8f56-21e6e418d756"/>
Assuming that "use cycle routes" is inherent in the Bike profile - but obviously this isn't the case:
<img width="200" src="https://github.com/user-attachments/assets/7f40c988-0a3a-4de4-910e-84a9d0b0bf0f"/> . . . <img width="200" src="https://github.com/user-attachments/assets/d8f5d21d-4a8f-4553-be8d-7857de62cf32"/> 
Between (4) and (5) the street is used instead. No big deal - I can live with using the Mountainbike profile.
Here the files: [Test cycle routes 1.zip](https://github.com/user-attachments/files/16230545/Test.cycle.routes.1.zip)

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for opening this discussion.

- Cruiser "Bike" uses the BRouter [fastbike](https://github.com/abrensch/brouter/blob/master/misc/profiles2/fastbike.brf) profile
- Cruiser "Mountain bike" uses the BRouter [trekking](https://github.com/abrensch/brouter/blob/master/misc/profiles2/trekking.brf) profile

Please check their links, they are also used in [BRouter website](https://brouter.de/brouter-web/).

Should we use the BRouter "trekking" profile for both "Bike" and "Mountain bike"?

**Reply by 0709wiwiwi** [reply]
To quickly check what is hidden behind track or routes display, the following tool would be very useful and thus make the very nice desktop version even more valuable.. See: https://github.com/devemux86/cruiser/discussions/309
![Check map background](https://github.com/user-attachments/assets/295a2cb3-b994-4c75-9e81-21c9a8cf76c0)

**Comment by devemux86** [comment]
Next Cruiser will use the BRouter trekking and options for all bike profiles.

**Comment by devemux86** [comment]
Cruiser now uses the BRouter trekking and options for all bike profiles:

- https://github.com/devemux86/cruiser/discussions/610

**Comment by devemux86** [comment, marked as answer]
- BRouter trekking profile is used for all app bike profiles in https://github.com/devemux86/cruiser/discussions/641
